src/routes/user.js

```diff
@@ -23,10 +24,20 @@ router.get('/breaches', requireSessionUser, breachesPage)
 // data removal page
 router.get('/data-removal', requireSessionUser, dataRemovalPage)
 
+// emails page
+router.get(['/emails', '/emails/:template'], requireSessionUser, emailsPage)
+
 // settings page
 router.get('/settings', requireSessionUser, settingsPage)
 
 // sign the user out
 router.get('/logout', asyncMiddleware(logout))
 
+// send test email
+router.post(
+  '/send-test-email',
+  requireSessionUser,
+  asyncMiddleware(sendTestEmail)
+)
```

Comment:
Do we have anything that allows us to distinguish between local and other environments? I think we might be setting ourselves up for a DoS attack if we leave this open on public environments?

Reply:
Right! @rhelmer added rate limiting for sending email verifications in PR #2807 — could we adopt this mechanic to work for this use-case easily?

Reply:
We may want to consider restricting this further - I noticed there's this entry in the `.env-dist`:
```
# Email addresses that are allowed to test and send emails
ADMINS=
```

Reply:
> We may want to consider restricting this further - I noticed there's this entry in the `.env-dist`:
> 
> ```
> # Email addresses that are allowed to test and send emails
> ADMINS=
> ```

Would it be possible to restrict this API to only be used by authenticated users that are on that list?

Reply:
Yes, the send functionality should be restricted to admins (dev and QA) only.  For context, this is the reason @flodolo requested a separate version without auth, so that volunteers could preview without needing admin/send access

Reply:
yes so in the v1 code, we have a middleware`requireAdminUser` which is used on all email testing endpoints